README.md

```diff
@@ -0,0 +1,229 @@
+# Ares
+
+Ares is an application server for the [Microgram](https://github.com/zalora/microgram). We can use it to control different applications (for example, start and stop them, or get logs information) via simple commands.
+
+Ares uses two concepts:
+
+1. Application. There're many different applications we can work with via Ares.
+2. Service. Ares supports only two services: [Angel](https://github.com/zalora/angel), daemon for the process monitoring/management, and [Nginx](https://github.com/zalora/microgram/tree/master/pkgs/nginx).
+
+## Running
+
+Basic usage is:
+
+```bash
+$ ares CONFIG
+```
+
+where `CONFIG` is a path to configuration file. Example of configuration file:
+
+```json
+{
+  "angelPath": "/nix/store/28v1rlns1cnd2i0p9zyg4wx5rpdvih2m-angel-0.5.1/bin/angel",
+  "appsDataDirRoot": "/var/apps",
+  "builtinApps": {},
+  "dataDir": "/var/ares",
+  "nginxConfigFile": "/nix/store/7l4kgz8m0jh6l517jsy5c5fq45ivlrjd-ares-nginx.conf",
+  "nginxEnable": true,
+  "nginxPath": "/nix/store/bik80155gfarkx1ggakl7ykl5n37wldf-nginx-1.9.4/bin/nginx",
+  "nginxRestartOnFailureDelay": 3000000,
+  "nixEnvPath": "/nix/store/fnky09kvm0rm2ps9msiy6cznvnfxflbq-nix-1.10/bin/nix-env",
+  "port": 13,
+  "profilesDir": "/nix/var/nix/profiles/ares-apps",
+  "runDir": "/run/apps/ares",
+  "wtfdbFile": "/var/ares/wtfdb.json"
+}
+```
+
+## API
+
+Ares listens UNIX-socket, not TCP. You can use `curl` to make requests, for example via such a script:
+
+```bash
+curl -fsS --unix-socket "$RUN_DIR"/warp.sock -X "$METHOD" http:"$URL" "$ADDITIONAL_ARGS" | jq .
+```
+
+where:
+
+- `$RUN_DIR` - path to directory where `ares` started from.
+- `$METHOD` - request method, like `GET`, `PUT` or `DELETE`.
+- `$URL` - request URL, like `/stop` or `/logs`.
+- `$ADDITIONAL_ARGS` - additional arguments for `curl`.
+
+### Start
+
+Starts `ares` as a daemon.
+
+Command:
+
+```bash
+$ nohup ares CONFIG 0<&- 1>ares.stdout.log 2>ares.stderr.log &
```

Comment:
I took it from here: https://github.com/zalora/eris/blob/master/apps/ares/default.nix#L75